test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7434,6 +7434,13 @@ def forward(self, x):
         x = torch.tensor([[True, False], [False, False]])
         self.run_test(M(), (x, ))
 
+        class MDim(torch.nn.Module):
+            def forward(self, x):
+                return x.all(dim=1)
```

Comment:
nit: do we need to test the two options of `keepdim` too?

Reply:
Added tests